Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -806,68 +850,72 @@ def ignore_property(name, ignore_regex):
         if _is_workspace_property(prop):
             value_str = prop.valueAsStr
             try:
-                retvals[name]=_api.AnalysisDataService[value_str]
+                retvals[name] = _api.AnalysisDataService[value_str]
             except KeyError:
                 if not prop.isOptional():
-                    raise RuntimeError("Internal error. Output workspace property '%s' on algorithm '%s' has not been stored correctly."
+                    raise RuntimeError("Internal error. Output workspace property '%s' on "
+                                       "algorithm '%s' has not been stored correctly."
                                        "Please contact development team." % (name,  algm_obj.name()))
         else:
             if hasattr(prop, 'value'):
-                retvals[name]=prop.value
+                retvals[name] = prop.value
             else:
-                raise RuntimeError('Internal error. Unknown property type encountered. "%s" on algorithm "%s" is not understood by '
-                       'Python. Please contact development team' % (name, algm_obj.name()))
-
-
+                raise RuntimeError('Internal error. Unknown property type encountered. "%s" '
+                                   'on algorithm "%s" is not understood by '
+                                   'Python. Please contact development team' % (name, algm_obj.name()))
     nvals = len(retvals)
     nlhs = lhs[0]
-    if nlhs > 1 and nvals != nlhs:
+
+    # If we have more than one value but not the same number of values throw
+    if 1 < nlhs != nvals:
```

Comment:
Can we put this back to what it was originally: `if nlhs > 1 and nvals != nlhs:` ?